### ISSUE TITLE
Warnings Not Annotated

### ISSUE THREAD

**Issue by jherdman** [xt0rted/stylelint-problem-matcher#170]
I'm in the middle of introducing Stylelint to a project. In order to smooth things over a little I've set `defaultSeverity` to "warning", which reports as below, but doesn't issue an error code:

```
app/styles/utilities/shadow.css
  8:35  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
  8:37  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
  8:39  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 12:35  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 12:37  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 12:39  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 16:33  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 16:35  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 16:37  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 20:33  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 20:35  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
 20:37  ⚠  Expected single space after "," in a single-line function   function-comma-space-after
```

I'm not seeing these annotations in my build, which would be desirable to encourage teammates to resolve these. Is there a configuration option I'm missing with this action?

**Comment by andresriveratoro** [comment]
Are there any plans on adding the option to create annotations on warning?

**Comment by xt0rted** [comment]
I just tried implementing this and ran into issues where if your build output contains errors and warnings then only the first severity level gets matched. This looks to be a limitation of problem matchers so I need to look into this more to see if there's some way to make this work. I also discovered that enabling the env var `FORCE_COLOR` breaks problem matchers like this one so that needs some additional looking into as well.
